microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -17,41 +17,33 @@ Publishing your Progressive Web App (PWA) to the [Microsoft Store](/windows/uwp/
 | **Discoverability** | Users naturally look for apps in the app store.  When you publish to the Microsoft Store, millions of Windows users can discover your PWA alongside other Windows apps.  The Store showcases apps through categories, curated collections, and more.  App discovery portals provide an easy browsing and shopping experience for potential users of your app.  You can even enhance your Store listing with screenshots, a hero image, and video trailers - see [App screenshots, images, and trailers](/windows/apps/publish/publish-your-app/screenshots-and-images). |
 | **Trustworthiness** | Windows customers know they can trust their Microsoft Store purchases and downloads, because they adhere to the rigorous Microsoft [quality and safety standards](/legal/windows/agreements/store-policies). |
 | **Easy install** | The Microsoft Store provides a consistent and user-friendly install experience across [all Windows 10 or later apps](https://www.microsoft.com/store/apps/windows). |
-| **App analytics** | The [Windows Partner Center dashboard](/windows/uwp/publish/index) provides you with [detailed analytics](/windows/uwp/publish/analytics) about your app health, usage, and more. |
+| **App analytics** | The [Microsoft Partner Center dashboard](/windows/uwp/publish/index) provides you with [detailed analytics](/windows/uwp/publish/analytics) about your app health, usage, and more. |
 
 To publish your PWA to the Microsoft Store, no code changes are required.  Instead, you create an app reservation, package your PWA, and submit your package to the Store.  The following sections explain the steps.
 
 
 <!-- ====================================================================== -->
 ## Create an app reservation
 
-[Windows Partner Center](https://partner.microsoft.com/dashboard/windows/overview) is the hub for you to submit your app to the Microsoft Store.
+[Microsoft Partner Center](https://partner.microsoft.com/dashboard/windows/overview) is the hub for you to submit your app to the Microsoft Store. 
 
 To create an app reservation:
 
-1.  To display your enrolled programs:
-    1.  Sign into Windows Partner Center with your Microsoft account and go to the [Partner Center Dashboard](https://partner.microsoft.com/dashboard/home).
-    1.  Navigate to **Windows & Xbox**.
-        *   If **Windows & Xbox** is displayed, your app is already enrolled.
-        *   If **Windows & Xbox** isn't displayed, select **Add program**.
+1.  Before you can create an app reservation, you must have a **personal** Microsoft account that is enrolled in the Windows Developer Program.
```

Comment:
```suggestion
1.  Before you can create an app reservation, you must have a **personal** Microsoft account (not a work Microsoft account or a school Microsoft account) that is enrolled in the Windows Developer Program.
```
instead of emphasizing "personal", more helpful and specific is to explicitly state what account types not to use.

wording derived from step 4 of
https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/aad-account#if-the-primary-owner-of-your-partner-center-account-isnt-a-microsoft-account-msa